core/src/main/java/com/nuodb/migrator/backup/loader/LoadTableForkWork.java

```diff
@@ -112,6 +114,26 @@ public void execute() throws Exception {
                 backupLoaderManager.afterLoadRow(this, loadTable, row);
             }
             commitExecutor.finish();
+
+            // added to alter the table after loading the data
+            try {
+                // added gen always fix
+                GlobalStore global = GlobalStore.getInstance();
+                CommitStrategy commitStrategy = backupLoaderContext.getCommitStrategy() != null
+                        ? backupLoaderContext.getCommitStrategy()
+                        : new BatchCommitStrategy();
+
+                String alterScript = global.alterScript(loadTable);
+                if (alterScript != null) {
+                    PreparedStatement newStatement = getSession().getConnection().prepareStatement(alterScript);
+                    commitAlter = commitStrategy.createCommitExecutor(newStatement, null);
+
+                    commitAlter.execute();
+                    commitAlter.finish();
+                }
+            } catch (Exception npe) {
```

Comment:
Same comment here about exception handling.

Reply:
Adding Log4j to log the logs in the file